### PR TITLE
Rename GamerOS to ChimeraOS

### PR DESCRIPTION
**Pull request by Samsagax**
Rename all artifacts to the new name

~~This change includes the nmtui integration described in #34~~

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 temp/
 output/
-gameros/extra_pkg/
-gameros/work/
-gameros/pacman.conf
+chimeraos/extra_pkg/
+chimeraos/work/
+chimeraos/pacman.conf
+temp/
```
Comment (alkazar) [inline .gitignore:6]: temp is listed twice

File: chimeraos/profiledef.sh [hunk 2]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2034
 
-iso_name="gameros"
-iso_label="GAMEROS_$(date +%Y%m)"
-iso_publisher="GamerOS <https://github.com/gamer-os/gamer-os>"
-iso_application="GamerOS Installer"
+iso_name="chimeraos"
+iso_label="CHIMERAOS_$(date +%Y%m)"
+iso_publisher="ChimeraOS <https://github.com/ChimeraOS/chimera>"
```
Comment (alkazar) [inline chimeraos/profiledef.sh:6]: The link should probably be https://github.com/ChimeraOS

**Comment by alkazar** [comment]
We should probably also replace or maybe remove the splash screen which still has the old GamerOS logo.

**Comment by Samsagax** [comment]
> We should probably also replace or maybe remove the splash screen which still has the old GamerOS logo.

I didn't know that splash existed. Or you mean the Syslinux boot prompt?

**Comment by alkazar** [comment]
It is the splash screen used for the install image. I'll make a replacement one.

File: chimeraos/profiledef.sh (hunk 2 above)
Comment (Samsagax) [reply chimeraos/profiledef.sh]: Done

File: .gitignore (hunk 1 above)
Comment (Samsagax) [reply .gitignore]: Removed duplicated

**Comment by Samsagax** [comment]
> It is the splash screen used for the install image. I'll make a replacement one.

Should I change something? Or you handle it separately?

**Comment by alkazar** [comment]
I'll just add it in a separate change.

**Comment by Samsagax** [comment]
> I'll just add it in a separate change.

Great, then. I've just rebased and pushed this branch so it's clean for merge.

**Comment by alkazar** [comment]
Thanks!

**Pull request merged**